recipe/set_bld_opts.bat

```diff
@@ -45,6 +45,9 @@ set BLD_OPTS=%WIN64% ^
     HDF4_LIB="%LIBRARY_LIB%\hdf.lib %LIBRARY_LIB%\mfhdf.lib %LIBRARY_LIB%\xdr.lib" ^
     HDF5_DIR=%LIBRARY_PREFIX% ^
     HDF5_LIB=%LIBRARY_LIB%\hdf5.lib ^
+    TILEDB_ENABLED=YES ^
+    TILEDB_DIR=%LIBRARY_PREFIX% ^
+    TILEDB_LIB=%LIBRARY_LIB%\tiledb.lib ^
```

Comment:
Should be:
```
TILEDB_ENABLED=YES ^
TILEDB_CFLAGS=-I%LIBRARY_INC% ^
TILEDB_LIBS=%LIBRARY_LIB%\tiledb.lib ^
```
Note: LIB**S** not LIB. see https://github.com/OSGeo/gdal/blob/master/gdal/nmake.opt#L688-L691

Reply:
Turns out that `TILEDB_LIBS` hasn't been added to `EXTERNAL_LIBS` in `nmake.opt` so that needs to be patched... Might be worth sending in a PR to `gdal` since it hasn't been fixed in their git.

Reply:
I see, thanks @gillins for you help on this.  We should be able to get another patch in (the osx issue seems to be fixed).

Reply:
https://github.com/OSGeo/gdal/pull/1644